specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ManagedDatabaseSchema.json

```diff
@@ -0,0 +1,648 @@
+          }
+        },
+        "x-ms-examples": {
+          "Get managed database table": {
+            "$ref": "./examples/ManagedDatabaseTableGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedDatabaseColumnProperties": {
+      "description": "Managed database column properties.",
+      "type": "object",
+      "properties": {
+        "columnType": {
+          "description": "The table columns.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {}
+    },
+    "ManagedDatabaseColumn": {
+      "description": "A managed database column resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ManagedDatabaseColumnProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ManagedDatabaseColumnListResult": {
+      "description": "A list of managed database columns.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedDatabaseColumn"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedDatabaseSchemaProperties": {
+      "description": "Managed database schema properties.",
+      "type": "object",
+      "properties": {
+        "tables": {
+          "description": "The schema tables.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedDatabaseTable"
```

Comment:
including payload of child resources in the payload of parent is not allowed. Following reasons -
1. Breaks RBAC
2. Response payload can run into the ARM 8 MB limit and break customers
3. Provides sub-optimal paging since the standard nextLink which ARM supports at top level will not work
Please exclude it from all resource responses.